packages/simcore-sdk/src/simcore_sdk/node_data/data_manager.py

```diff
@@ -129,18 +130,20 @@ async def _pull_file(
     *,
     io_log_redirect_cb: LogRedirectCB | None,
     save_to: Path | None = None,
+    r_clone_settings: RCloneSettings | None,

```

Comment:
Q: if r_clone is used only for folders, why is a parameter here? I suggest to wait until rclone is also used to upload files to add this option (and then we make it required)